### PR TITLE
upgrade to Antora 2 and switch to Jenkins pipeline

### PR DESCRIPTION
**Pull request by mojavelinux**
* upgrade Netlify build (deploy previews) to Antora 2
* remove workaround that injects GITHUB_TOKEN into playbook in Netlify build
* configure Jenkins pipeline for production build
* upgrade Jenkins build to Antora 2
* create custom Docker image based on Antora image for use in Jenkins job
* switch builds to Node 10
* populate GIT_CREDENTIALS environment variable in Jenkins job

### CODE REVIEW

**Comment by mojavelinux** [comment]
cc: @amarantha-k Once I confirm the Netlify build succeeds, I feel confident this is ready. After merging, I'll decommission the old Jenkins job and activate the new one (docs.couchbase.com-pipeline).

I'll follow-up shortly with instructions for writers for switching to Antora 2.

**Comment by mojavelinux** [comment]
You will notice a few more warnings ;)

File: Dockerfile.ci [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This image extends the offical Antora image to add bash and the aws command. It's used by the CI job.
```
Comment (jamiltz) [inline Dockerfile.ci]:
```suggestion
# This image extends the official Antora image to add bash and the aws command. It's used by the CI job.
```

Comment (mojavelinux) [reply Dockerfile.ci]: Thanks James.

**Pull request merged**